Add stack 1.9.3 to v2 image

onslaughtq opened the pull request:
A version of the v2 image with stack 1.9.3 has been added,
and the README has been updated to reflect this as well.

onslaughtq committed:
Add stack 1.9.3 to v2 image

A version of the v2 image with stack 1.9.3 has been added,
and the README has been updated to reflect this as well.

qxjit commented on v2/1.9.3/Dockerfile:
```diff
@@ -0,0 +1,30 @@
+    curl -fSL https://github.com/commercialhaskell/stack/releases/download/v$STACK_VERSION/stack-$STACK_VERSION-linux-x86_64-static.tar.gz.asc -o stack.tar.gz.asc && \
+# setup gpg and get keys
+    export GNUPGHOME="$(mktemp -d)" && \
+    gpg --keyserver ipv4.pool.sks-keyservers.net --recv-keys C5705533DA4F78D8664B5DC0575159689BEFB442 && \
```
This doesn't build for me at the moment, and I believe it's because `--batch` needs to be added to this line to work around upstream `gpg` breakage. I believe we are feeling the impact of:

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=913614

Which is referenced from here:

https://github.com/nodejs/docker-node/issues/922

where they are getting the same `gpg: cannot open '/dev/tty': No such device or address` error that I am seeing.

qxjit commented:
@onslaughtq Can you try the workaround suggested in the linked items (adding `--batch`) and see if it will build? I'd like to record the problem and fix as part of this PR, but I can't push to your fork to make it appear here.

onslaughtq committed:
Fix for "fixed" gpg command

Due to a side-effect of a bug-fix in the gpg command, it is now
necessary to pass --batch to gpg commands that you wouldn't expect
to allocate a /dev/tty device.

In general though, passing --batch to gpg commands run within a script
is probably a good practice to follow.

onslaughtq commented:
Passing --batch to the gpg command has now resolved this issue and the image can be built again

qxjit reviewed (approved):
I was able to build an image and the orville tests are passing on all lts versions that orville currently builds with. I'll push the image to docker hub and merge this.
